docker-compose.yml

```diff
@@ -105,3 +105,20 @@ services:
         retries: 6
     ports:
       - "1434:1433"
+
+  init-service:
+      build:
+        context: .
+        dockerfile: init-service/Dockerfile
+        args:
+          SVC: init-service
+      image: ghcr.io/ndph-arts/init-service:${GITHUB_SHA}
+      restart: on-failure
+      environment:
+        ROLE_SERVICE: "http://role-service:82"
+        SITE_SERVICE: "http://site-service:83"
+        PRACTITIONER_SERVICE: "http://practitioner-service:80"
+        CONFIG_JSON: config-init.json
```

Comment:
this is temporary until you have the config server?

Reply:
Yes, it getting it from https://github.com/NDPH-ARTS/global-trial-config/blob/main/config-init.json